mapie/regression/regression.py

```diff
@@ -609,6 +622,17 @@ def predict(
                 - [:, 0, :]: Lower bound of the prediction interval.
                 - [:, 1, :]: Upper bound of the prediction interval.
         """
+
+        if (len(predict_params) > 0 and hasattr(self, '_predict_params') and
```

Comment:
Why are we checking for `hasattr(self, '_predict_params')`?

Reply:
It seems that I no longer need it following the modification I made to the nature of predict_params, which has become a boolean. Thank you for bringing up this point !